app/serializers/serialize_submission.rb

```diff
@@ -19,6 +19,10 @@ def call
         test_run: Exercism::Routes.api_submission_test_run_url(
           submission.uuid,
           auth_token: user.auth_tokens.first.to_s
+        ),
+        files: Exercism::Routes.api_solution_original_solution_files_url(
```

Comment:
```suggestion
        initial_files: Exercism::Routes.api_solution_original_solution_files_url(
```

Reply:
Should this not be on the solution, not the submission? This doesn't feel like a property of a submission to me, it feels like a property of the whole solution.

Reply:
I agree. We pass in the submission to the editor, and not the solution, though. I'll open an issue to look into passing the whole solution instead.